src/components/sidenav/demoBasicUsage/index.html

```diff
@@ -4,11 +4,10 @@
   <section layout="row" flex>
 
     <md-sidenav
-        class="md-sidenav-left"
+        class="md-sidenav-left md-whiteframe-4dp"
```

Comment:
Not sure, why you've changed the directive to the class.

`md-whiteframe="4"` should work fine and also shows the API in use.